mne/beamformer/_lcmv.py

```diff
@@ -398,6 +398,51 @@ def apply_lcmv_raw(raw, filters, start=None, stop=None, max_ori_out='signed',
     return next(stc)
 
 
+@verbose
+def apply_lcmv_cov(data_cov, filters, verbose=None):
+    """Apply Linearly Constrained  Minimum Variance (LCMV) beamformer weights.
+
+    Apply Linearly Constrained Minimum Variance (LCMV) beamformer weights
+    to a data covariance matrix to estimate source power.
+
+    NOTE : This implementation has not been heavily tested so please
+    report any issue or suggestions.
```

Comment:
I don't think we need this note since you test against FT output (and it's a refactoring of Marijn's code already). Feel free to remove all such warnings while you're at it.

Reply:
I removed the one still there for the overhauled LCMV code in ``apply_lcmv_raw``. I left the ``tf_lcmv`` ones untouched, I guess, that is still "not heavily tested" ... We should decide what to do with this code, anyway!